config/src/config/logger_config.rs

```diff
@@ -10,13 +12,36 @@ pub struct LoggerConfig {
     pub is_async: bool,
     // chan_size of slog async drain for node logging.
     pub chan_size: usize,
+    // The default logging level for slog.
+    #[serde(serialize_with = "serialize_filter_level")]
+    #[serde(deserialize_with = "deserialize_filter_level")]
+    pub filter_level: FilterLevel,
 }
 
 impl Default for LoggerConfig {
     fn default() -> LoggerConfig {
         LoggerConfig {
             is_async: true,
             chan_size: 256,
+            filter_level: FilterLevel::Info,
         }
     }
 }
+
+fn serialize_filter_level<S>(filter_level: &FilterLevel, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    let filter_level_string = filter_level.as_str().to_string();
+    filter_level_string.serialize(serializer)
+}
+
+fn deserialize_filter_level<'de, D>(deserializer: D) -> Result<FilterLevel, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let filter_level_string: String = Deserialize::deserialize(deserializer)?;
+    let filter_level = FilterLevel::from_str(&filter_level_string)
+        .expect("Tried to deserialize an invalid FilterLevel");
```

Comment:
This shouldn't be an expect but rather an error. You may need to map_err. There are some examples in code.